src/client.js

```diff
@@ -139,16 +146,16 @@ export default class DockerExecWebsocketClient extends EventEmitter {
   }
 
   //pauses input coming in from server, useful if you're running out of memory on local
-  pause () {
+  pause() {
     this.sendCode(msgcode.pause);
   }
 
   //analogue of pause
-  resume () {
+  resume() {
```

Comment:
if  `this.stderr.write` returns false then we pause... and the when it emits `drain` then we unpause...
Note that we only have one pause/resume covering both of these... so we have to have a boolean and track the state of both of these...